dali/python/nvidia/dali/backend.py

```diff
@@ -28,11 +29,13 @@
     Init(OpSpec("CPUAllocator"), OpSpec("PinnedCPUAllocator"), OpSpec("GPUAllocator"))
     initialized = True
 
-    # pybind11 deprecations
-    def asCPU(self):
-        warnings.warn("asCPU is deprecated since v0.7, please use as_cpu", DeprecationWarning, stacklevel=2)
-        return self.as_cpu()
-    TensorListGPU.asCPU = asCPU
+    # py27 deprecation
+    if sys.version_info[0] < 3:
+        warnings.simplefilter("default", append=True)
```

Comment:
how about a more custom message. Saying that DALI won't be supported for python 2.7 after release X

Reply:
I followed the messaging from pip. I want to emphasis the fact that Python 2.7 won't be maintained any longer first and then, as a result, DALI will do the same.
Can you suggest the desired wording?

Reply:
For example, Librosa says: ```FutureWarning: You are using librosa with Python 2. Please note that librosa 0.7 will be the last version to support Python 2, after which it will require Python 3 or later.```